.github/workflows/build_ncs.yml

```diff
@@ -0,0 +1,64 @@
+name: Build binary for Nordic NCS
+
+# Runs when pushed to main, semver tagged release or any PR
+on:
+  push:
+    branches: [ main ]
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Do we need to build binaries on every PR/merge? To me it makes a ton of sense to generate (and upload) them when a release is created as there's a lot of room for human error in that process. Maybe a manual trigger (`workflow_dispatch:`) for the other times?

Reply:
Definitely don't _need_. It's a nice benefit for other testers and it's cheap/free. The downside is more chances for failed runs, resulting in annoying notifications.

Happy to change the behavior, even if we want to start simple  initially to test things out!

Reply:
Moved to only manual for now!